public_html/css/main.css

```diff
@@ -124,6 +125,10 @@ button {
 	color: red;
 }
 
-.error-message {
-	width: 100px;
+.error_hidden {
```

Comment:
So that it would be obvious from the html code that the error is hidden. I mean, it's not like we spend any resources for this